### PR TITLE
Added user model

### PR DESCRIPTION
**Pull request by Hogbunike**
#2

### CODE REVIEW

File: backend/users/models.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    is_staff = models.BooleanField(default=False)
+    date_joined = models.DateTimeField(default=timezone.now)
+
+    date_of_birth = models.DateField(null=True, blank=True)
+    first_name = models.CharField(max_length=30, blank=True)
+    last_name = models.CharField(max_length=30, blank=True)
+    gender = models.CharField(max_length=10, blank=True)
+    description = models.TextField(blank=True)
+    address = models.TextField(blank=True)
+    image = models.ImageField(upload_to='images/', blank=True, null=True)
```
Comment (denscholar) [inline backend/users/models.py]: Nice job!!!! However, I supposed this is the user image, like the profile image. Please, this should come in the profile model. Kindly refactor for approval. Thanks very much.

**Review by denscholar — CHANGES_REQUESTED** [review]
This is good but please consider my feedback and make adjustments. I love what you are doing. Keep it up.

**Comment by MichealCodez** [comment]
Kindly make changes so we can continue with #3 
Thanks man

**Review by denscholar — APPROVED** [review]
Weldone sir. You have done and I have nothing else to say other than you should merge your code.

**Pull request merged**